### PR TITLE
First pull request

### PR DESCRIPTION
**Pull request by jeevankishor**
Redone - Commits

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,340 +0,0 @@
-                    GNU GENERAL PUBLIC LICENSE
```
Comment (SriramKeerthi) [inline LICENSE:1]: Licenses aren't necessary right now

**Comment by SriramKeerthi** [comment]
Eclipse files .classpath, .project and the folder .settings can be safely ignored when checking into Github. In a live project, add these to the .gitignore as well. Not necessary now.

**Comment by SriramKeerthi** [comment]
The idea of .gitignore is to not commit unnecessary files. Binary files are usually not necessary. Adding bin/ to your .gitignore is a good idea too.

**Comment by SriramKeerthi** [comment]
There seems to be some kind of issues with your PR. Pull the master branch into your development branch and resolve conflicts.

**Comment by jeevankishor** [comment]
Noted. Will do .

**Pull request closed without merging**